starters/next-drupal-starter/pages/examples/index.jsx

```diff
@@ -0,0 +1,48 @@
+import Head from "next/head";
+import Link from "next/link";
+import Layout from "../../components/layout";
+
+export default function AuthApiExampleTemplate({ articles, footerMenu }) {
```

Comment:
Update name for this template
```suggestion
export default function ExamplesPageTemplate({ articles, footerMenu }) {
```